src/DDTrace/OpenTelemetry/SpanContext.php

```diff
@@ -23,35 +24,31 @@ final class SpanContext implements SpanContextInterface
 
     private bool $remote;
 
-    private ?string $traceId;
-
     private ?string $spanId;
 
-    private bool $isValid = true;
+    private static function getRootSpan(SpanData $span): RootSpanData
+    {
+        while (!($span instanceof RootSpanData) && $span->parent) {
+            $span = $span->parent;
+        }
+        return $span;
+    }
```

Comment:
Note to self: May be wrong when considering intertwined `start_trace_span` & OTel starts w/ `setParent(false)`